src/features/mentors-wanted/mentors-wanted.module.scss

```diff
@@ -0,0 +1,63 @@
+.mentors-wanted {
+  display: flex;
+  gap: 20px;
+  background-color: $color-yellow-50;
+
+  .content-left {
+    display: flex;
+    flex-direction: column;
+    gap: 40px;
+
+    max-width: 43%;
+
+    text-align: left;
+  }
+
+  .link {
+    font-size: 18px;
+    text-decoration: underline;
+  }
+
+  .content {
+    display: flex;
+    align-items: start;
+    justify-content: space-between;
+    padding: 80px 120px;
+
+    p {
+      max-width: 825px;
+      font-size: 18px;
+      line-height: 28px;
+    }
+  }
+
+  .picture {
+    align-self: center;
+    max-width: 290px;
+
+    img {
+      width: 100%;
+      height: 100%;
+      object-fit: contain;
+    }
+  }
+
+  @media (width <= 1440px) {
+    .content {
+      padding: 80px 40px;
+    }
+  }
+
+  @media (width <= 810px) {
```

Comment:
Please use mixins such as `@mixin media-tablet` instead of specifying values explicitly